### PR TITLE
Use softprops/action-gh-release in SwiftFormat.yml.

### PR DESCRIPTION
**Pull request by hjyamauchi**
Also tweak the swift versioin number in the release name.

### CODE REVIEW

File: .github/workflows/SwiftFormat.yml [hunk 1]
```diff
@@ -60,28 +63,24 @@ jobs:
...
-        id: create_release
+
+      # Prepare the path to the msi file with forward slashes because action-gh-release
+      # doesn't accept paths with backslashes.
+      - name: prepare msi path
+        run: |
+          set MSI_PATH=${{ github.workspace }}\artifacts\SwiftFormat.msi
+          set MSI_PATH=%MSI_PATH:\=/%
+          echo %MSI_PATH%
+          echo MSI_PATH=%MSI_PATH%>> %GITHUB_ENV%
```
Comment (compnerd) [inline .github/workflows/SwiftFormat.yml:74]: Why not use `cygpath -m`?

Comment (hjyamauchi) [reply .github/workflows/SwiftFormat.yml]: Done

File: .github/workflows/SwiftFormat.yml [hunk 2]
```diff
@@ -18,12 +18,15 @@ jobs:
...
     steps:
+      # Extract swift version (e.g., "swift-5.8-release" -> "5.8")
       - name: extract swift version
         run: |
           set SWIFT_VER=${{ matrix.swift_branch }}
           set SWIFT_VER=%SWIFT_VER:swift-=%
           set SWIFT_VER=%SWIFT_VER:-release=%
           echo %SWIFT_VER%
+          echo SWIFT_VER=%SWIFT_VER%>> %GITHUB_ENV%
+        shell: cmd
```
Comment (compnerd) [inline .github/workflows/SwiftFormat.yml:29]: Might be nice to convert this into a PowerShell script?

**Review by compnerd — APPROVED** [review]
Thanks, this looks great!

**Pull request merged**